lib/os.js

```diff
@@ -61,7 +82,4 @@ exports.getNetworkInterfaces = internalUtil.deprecate(function() {
 
 exports.EOL = isWindows ? '\r\n' : '\n';
 
-if (binding.isBigEndian)
-  exports.endianness = function() { return 'BE'; };
-else
-  exports.endianness = function() { return 'LE'; };
+binding.isBigEndian == true ? exports.endianness = function() { return 'BE'; } :  exports.endianness = function() { return 'LE'; };
```

Comment:
I don't think this needs changing. Not only does putting it all on one line make it exceed the 80 column line limit, but it makes it less readable.